Unix/create_user.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+#
+# Script to create user account and password for use by OMI
+# This file should be executed via ".", as it sets environment variables
+
+username=$OMI_USER
+
+if [ "x$username" = "x" ]; then
+    echo "Creating new user..."
+    username=omi_$USER
+fi
+
+egrep "^$username" /etc/passwd >/dev/null
+if [ $? -eq 0 ]; then
+    echo "User $username exists!"
+else
+    useradd=`which useradd`
```

Comment:
Isn't the location of useradd consistent across platforms (probably /usr/sbin)? Is there any system where this is NOT the case?

Also, what about Mac? Don't you want to bypass this on Mac OS?